.github/workflows/prerelease-exit.yml

```diff
@@ -0,0 +1,43 @@
+name: Exit Prerelease Mode
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Exit prerelease mode on release branch"
+        type: string
+        default: "main"
+        required: true
+
+jobs:
+  exit_prerelease:
+    name: Changesets Exit Prerelease
+    runs-on: ubuntu-latest
+    # Allow GITHUB_TOKEN to have write permissions
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout Repo
+        with:
+          submodules: recursive
+
+      - name: "Setup"
+        uses: ./.github/actions/setup
+        
+      - name: Generate gas reports
+        run: pnpm gas-report
+
+      - name: Outdated files, run `pnpm gas-report` and commit them
+        uses: ./.github/actions/require-empty-diff
+
+      - name: Remove pre.json
+        run: npx rimraf .changeset/pre.json
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Exit prerelease mode
+          # Commit these changes to the branch workflow is running against
+          branch: ${{ github.event.inputs.branch }}
```

Comment:
This is likely cleaner if we made this a pull request but don't have a strong preference either way.  The pull request could be automatically opened if it detects a pre.json instead of an action like here. THen we could delete this action file completely

Reply:
Pull request thinking about this now makes it more confusing since we would have pull requests doing all types of things

Reply:
agree with making it an action to avoid cluttering the PR